statsmodels/genmod/families/family.py

```diff
@@ -798,42 +806,35 @@ def deviance(self, endog, mu, scale=1.):
             Fitted mean response variable
         scale : float, optional
             An optional scale argument
-
         Returns
         --------
         deviance : float
             The deviance function as defined below
-
         Notes
         -----
         If the endogenous variable is binary:
-
         `deviance` = -2*sum(I_one * log(mu) + (I_zero)*log(1-mu))
-
```

Comment:
don't remove empty lines. 
Do you have an automatic reformatting setting in the editor?